index.ts

```diff
@@ -0,0 +1,65 @@
+"use strict";
+
+let onDisappearCallbacks: (() => void)[] = [];
+let onAppearCallbacks: (() => void)[] = [];
+let cursorIsVisible: boolean = true;
+
+const MousePresence : any = {
+    /**
+     * Add a callback for the mouse disappear event
+     * @Param {() => void} cb - The callback to be added
+     * @Return {() => void} A reference to the same callback
+     */
+    onDisappear: (cb: () => void): () => void => {
+        onDisappearCallbacks.push(cb);
+        return cb;
```

Comment:
Should we follow the pattern where callback queuer returns a deregisterer ?

Reply:
Sounds good, done